Makefile

```diff
@@ -31,10 +31,18 @@ deps_rh_centos:
 deps_fedora:
 	sudo dnf install $(RPMS)
 
-deps_debian:
+deps_debian: libjasper
 	sudo apt-get update
 	sudo apt-get install $(DEBS)
 
+libjasper:
+	IS_UBUNTU17=$(shell cat /etc/issue | grep -o "17")
+ifeq ($(IS_UBUNTU17), 17)
+	sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
```

Comment:
This is not cool for the developper to modify the apt configuration in a makefile. Creating a PR to disable JPEG2000 support instead.